Muvr/MKExerciseSession+MRManagedExerciseSession.swift

```diff
@@ -0,0 +1,15 @@
+import MuvrKit
+
+extension MKExerciseSession {
```

Comment:
It doesn't construct a managed object but uses a managed object to construct a ``MKExerciseSession``